CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```

Comment:
You do not need to add `## [Unreleased]` again ;] Just put it under the already existing one. 

More about format here: https://keepachangelog.com/en/1.0.0/